swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -642,6 +866,26 @@ protected void adjustForNullItem() {
 			}
 		}
 	}
+	
```

Comment:
Felt like this was better/safer outside of the Action Listener.

Reply:
I'm still trying to follow the Eclipse alphabetical approach from there code cleanup tools. Inner classes, declarations (alphabetical), methods (alphabetical). I realize you're not and for good reason in some places. I guess most IDE tools have some sort of field/method explorer.

Reply:
It was at first, but I adapted and I like the consistency.